### DISCUSSION TITLE
NPCs don't move until you approach them. Regular residents and minecolonies

### DISCUSSION THREAD

**Discussion by Osset1an** [IzzelAliz/Arclight#1590] [Q&A]
1.21.1 Neo. The Arclight core is where mods and plugins work together. There is a minecolonies mod. Before installing arclight, everything was fine, good simulation range, but after installation, apparently some kind of restriction was set, mobs that are a little further than one chunk from me do not move until I approach them, or they move but in very, very small jerks, what should I do? in server properties there is 32 range and 32 simulation. https://imgur.com/a/Fao7WMy

**Comment by qyl27** [comment]
Does increasing the simulation range helps?  

btw, if something not works as same as in the pure bukkit/forge/fabric/neoforge server, it should be consider as a bug. Feel free to submit an issue.

**Comment by cylian914** [comment]
this is a spigot feature (https://hub.spigotmc.org/stash/projects/SPIGOT/repos/spigot/browse/CraftBukkit-Patches/0011-Entity-Activation-Range.patch), i made a mod to basically remove this optimization (https://github.com/cylian914/fixAi, 1.20.1 compiled mod: https://github.com/cylian914/fixAi/releases/tag/1.0, other mc version*: https://github.com/cylian914/fixAi/releases/tag/1.0-dep_Remove)

*its the 1.20.1 version but i told it to try to load in any mc version, not guarantee to work
